src/main/proto/admin.proto

```diff
@@ -44,6 +44,7 @@ message CreateContextRequest {
   string name = 1;
   string replicationGroupName = 2;
   map<string, string> meta_data = 3;
+  bool dcbContext = 4;
```

Comment:
If I am not mistaken, the "meta_data" field here is rather configuration for the context, right? Couldn't the fact that it is a DCB context be part of that?

Reply:
@MGathier ?

Reply:
No preference here. Having it as a separate field makes it more explicite, but it may be redundant in the future.